exercises/kindergarten-garden/kindergarten_garden_test.py

```diff
@@ -26,16 +38,6 @@ def test_full_garden(self):
         self.assertEqual(
             garden.plants("Larry"), "Grass Violets Clover Violets".split())
 
-    def test_disordered_test(self):
-        garden = Garden(
-            "VCRRGVRG\nRVGCCGCV",
-            students="Samantha Patricia Xander Roger".split())
-        self.assertEqual(
-            garden.plants("Patricia"),
-            "Violets Clover Radishes Violets".split())
-        self.assertEqual(
-            garden.plants("Xander"), "Radishes Grass Clover Violets".split())
-
```

Comment:
I don't think there's a problem with having track-specific tests (as long as they do not conflict with canonical tests); there are other exercises in which we do this. I think we ought to keep this test, but add a comment that this is a track-specific test and is not canonical.